cloudinary_cli/core/admin.py

```diff
@@ -19,11 +19,16 @@
 @option("-o", "--optional_parameter", multiple=True, nargs=2, help="Pass optional parameters as raw strings.")
 @option("-O", "--optional_parameter_parsed", multiple=True, nargs=2,
         help="Pass optional parameters as interpreted strings.")
+@option("--recursive", nargs=1, help="Query a field in the response recursively.", default=None)
```

Comment:
Maybe it would be better to call it` auto_pagination` or `auto_paginate`(similar to what we have in search API). And why is it limited to only one field? What if someone needs multiple fields? or all fields? Once again, I would align it with search API, and make it a flag, and add another field called `filter_fields`